lib/params/RenderParams.cpp

```diff
@@ -338,6 +342,21 @@ void RenderParams::SetColorbarPbase(ColorbarPbase* pb){
 	
 }
 
+void RenderParams::_calculateStride( string varName ) {
```

Comment:
This `_calculateStride()` function seems to be duplicate to the one from `TFWidget.cpp`. Any specific reason to duplicate it in two classes?

Reply:
Hey Sam, there is a reason.  From my comment in this thread on December 19:

> Sam, your change requests are in. Note that there's a workaround in the RenderParams, which has duplicate code for calculating the stride (copied from TFWidget). This is going to be fixed during a refactorization of RenderParams (and maybe ParamsBase?) so that their responsibilities are lessened, but we're just a band-aid for now.

Reply:
Scott, I understand you have a work plan to resolve this duplication later on. But I don't understand *why* does it have to be like this at the first place.